### ISSUE TITLE
Ouroboros should be able to run the Python test suite

### ISSUE THREAD

**Issue by phildini** [pybee/ouroboros#11]
This ticket is really more of an "epic", to borrow an agile term.

We should get Ouroboros to the point where the stdlib parts of the cpython test suite can run against Ouroboros. This might mean borrowing stdlib implementations from other python implementations.

**Comment by Alexndrrr** [comment]
I agree that we should be able to run the Cpython test suite using the Ouroboros standard library, but I don't think this should count as "epic" in agile terms.

I think sooner rather than later we should figure out the arcane magic necessary to get the test suite to import ouroboros instead. That would be an invaluable for continuous integration. Whatever modules that aren't implemented yet in Ouroboros should fall back to the CPython version for testing, or better yet those tests should be skipped.
